src/compound/PositionsManager.sol

```diff
@@ -446,6 +446,7 @@ contract PositionsManager is IPositionsManager, MatchingEngine {
 
         _updateP2PIndexes(_poolToken);
         uint256 toWithdraw = Math.min(_getUserSupplyBalanceInOf(_poolToken, _supplier), _amount);
+        if (toWithdraw == 0) revert UserNotMemberOfMarket();
```

Comment:
I just realised that this line:
`if (!userMembership[_poolToken][_supplier]) revert UserNotMemberOfMarket();` is already doing the job and this why we did not implement it in Compound I think. 

Wdyt? Now I think it's really useless to add this check tbh

Reply:
I assumed that we saw the bug happening

>I understand that the following situation can occur: a user is member of a market, but cannot repay more than 0 (due to roundings ?). Maybe we can have a quick test checking that now it reverts as expected.

But if not (and we can prove it), then I agree that this is useless

Reply:
Shall I close the PR and issues then?

Reply:
I was against introducing this at first so I can't be against closing this now 🤭
(ie I am in favor of closing it)